src/TLuaInterpreter.cpp

```diff
@@ -346,8 +346,8 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
 {
     Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
     if( ! pHost ) {
-        lua_pushstring( L, tr( "raiseEvent: NULL Host pointer - something is wrong!" )
-                        .toUtf8().constData() );
+        lua_pushstring(L, QStringLiteral("raiseEvent: NULL Host pointer - something is wrong!")
+                       .toLatin1().constData());
```

Comment:
:blush: Humm, yes, that does seem to be the case for a fair proportion of the strings, gonna have to review what I dun!